### ISSUE TITLE
Gradle rule for PathSensitivity false positive

### ISSUE THREAD

**Issue by wakingrufus** [nebula-plugins/nebula-archrules#63]
```methods that annotated with Input file annotations and are declared in classes that are annotated with @CacheableTask should be annotated with @PathSensitive
Check failure: methods that annotated with Input file annotations and are declared in classes that are annotated with @CacheableTask should be annotated with @PathSensitive
Method <com.netflix.nebula.archrules.gradle.CheckRulesTask.getRulesClasspath()> is not annotated with @PathSensitive ```

in this case, the `@Classpath` annotation was used instead, which should be allowed
